Packs/CommonScripts/Scripts/CalculateEntropy/CalculateEntropy.py

```diff
@@ -0,0 +1,40 @@
+import demistomock as demisto
+from CommonServerPython import *  # noqa: E402 lgtm [py/polluting-import]
+from CommonServerUserPython import *  # noqa: E402 lgtm [py/polluting-import]
+import math
+import string
+
+
+def calculate_shannon_entropy(data, minimum_entropy):
+    """Algorithm to determine the randomness of a given data.
+    Higher is more random/complex, most English words will yield in average result of 3
+    Args:
+        data (str): The data to calculate entropy on.
+        minimum_entropy (float): The minimum entropy.
+    Returns:
+        (str, dict, dict). Human readable, context, raw response
+    """
+    entropy = 0.0
+    # each unicode code representation of all characters which are considered printable
+    for char in (ord(c) for c in string.printable):
+        # probability of event X
+        p_x = float(data.count(chr(char))) / len(data)
+        if p_x > 0:
+            entropy += - p_x * math.log(p_x, 2)
+    if entropy >= minimum_entropy:
+        return '', {'EntropyResult': {'checked_value': data, 'entropy': entropy}}, {}
```

Comment:
let's add md + and if the library @Itay4 mentioned is an easy change let's do that too

Reply:
@barchen1 just add the human readable here